test/tasks/logging.ts

```diff
@@ -0,0 +1,25 @@
+import { assert } from "chai";
+import Debug from "debug";
+
+const log = Debug("test:console:log");
+const warn = Debug("test:console:warn");
+
+let consoleLog: typeof console.log;
+let consoleWarn: typeof console.warn;
+let consoleSuppressed = false;
+
+export const useDebugConsole: () => void = () => {
```

Comment:
This is quite hacky (manipulating global state like this and relying that use/restore are always called in the right order). 
I'm not sure why it is needed and if the use case would allow passing a custom logger instance (which is how I think you would usually handle custom logging needs).

Reply:
I agree. It's used to suppress the output of the script in the tests to avoid having noisy console output. Since it's helpful to have some output for debugging, instead of suppressing it with `() => {}` I decided to use a debug logging function (activated with the env var as in the test plan).

I think replacing the log function in the withdraw script would require quite some work, most importantly finding out a good logging framework for it. Since it's only used in tests, I'd prefer merging this PR as it is and taking care of logging in a future PR.

Reply:
Maybe we can use `debug` everywhere in the tasks, and check if there is a way to change its output behaviour to only print the message without the namespace and timing.

Reply:
I wasn't able to manipulate how the ancillary info of `debug` is printed.